### ISSUE TITLE
Cache Invalidation 

### ISSUE THREAD

**Issue by simplyWiri** [KiameV/rimworld-rimfridge#51]
Hi,

Is this section of code correct? 
```cs
public override void PostDeSpawn(Map map)
{
    base.PostDeSpawn(map);
    FridgeCache.AddFridge(this, map); // Shouldn't this be `RemoveFridge`?
}
```

https://github.com/KiameV/rimworld-rimfridge/blob/master/Source/CompRefrigerator.cs#L211-L216

**Comment by KiameV** [comment]
Eww that looks like a nasty mistake. Let me dive into the code and check it out.

**Comment by KiameV** [comment]
Thanks for bringing this to my attention! Fixed with version 1.2.1.2
